Modules/ThirdParty/HDF5/.gitattributes

```diff
@@ -2,3 +2,5 @@ src/itkhdf5/src/H5C.c        hooks-max-size=500000
 src/itkhdf5/src/H5Dchunk.c   hooks-max-size=500000
 src/itkhdf5/src/H5Shyper.c   hooks-max-size=600000
 src/itkhdf5/src/H5Tconv.c    hooks-max-size=500000
+src/itkhdf5/src/H5Ppublic.h  hooks-max-size=500000
+src/H5Odtype.c               no-lf-at-eof
```

Comment:
This attribute did not work to remove the Ghostflow checker error. The error comes from the sub tree. Likely the attribute should be in the subtree not the main tree. I don't think it's worth the effort to redo the merge again to address this.